website_and_docs/content/documentation/webdriver/elements/finders.zh-cn.md

```diff
@@ -64,14 +61,12 @@ val vegetable: WebElement = driver.findElement(By.className("tomatoes"))
 {{< /tabpane >}}
 
 
-### Evaluating a subset of the DOM
+### 在部分DOM中查找元素
+
+以上代码无法获取在样例第二个列表中的"tomato"元素。如果知道目标元素所属位置且希望获取指定范围里的元素而不是整个上下文中的第一个，可以进行部分搜索。
```

Comment:
Hello,

If you mean `Rather than finding a unique locator in the entire DOM, it is often useful to narrow the search to the scope of another located element. ` 

I translated it as the sentence on line 66. The original sentence wants to say that "search in subset DOM" is better. I use the sentence `如果知道目标元素所属位置且希望获取指定范围里的元素而不是整个上下文中的第一个，可以进行部分搜索。` , which is more clear for Chinese audiences reading.

If you think it is better leaving a direct transaction, please let me know.

Thanks